transmogrifier/sources/datacite.py

```diff
@@ -282,41 +262,53 @@ def transform(self, xml: Tag) -> timdex.TimdexRecord:
                 subjects_dict.setdefault(subject.attrs["subjectScheme"], []).append(
                     subject.string
                 )
-        kwargs["subjects"] = [
+        fields["subjects"] = [
             timdex.Subject(value=value, kind=key)
             for key, value in subjects_dict.items()
         ] or None
 
         # summary, uses description list retrieved for notes field
-        kwargs["summary"] = [
+        fields["summary"] = [
             d.string
             for d in descriptions
             if d.get("descriptionType") == "Abstract" and d.string
         ] or None
 
-        # citation, generate citation from other fields
-        kwargs["citation"] = generate_citation(kwargs)
+        return fields
+
+    @classmethod
+    def get_main_titles(cls, xml: Tag) -> list[str]:
+        """
+        Retrieve main title(s) from a Datacite XML record.
+
+        Overrides metaclass get_main_titles() method.
 
-        return timdex.TimdexRecord(**kwargs)
+        Args:
+            xml: A BeautifulSoup Tag representing a single Datacite record in
+                oai_datacite XML.
+        """
+        return [t for t in xml.metadata.find_all("title", titleType=False)]
 
     @classmethod
-    def create_source_record_id(cls, xml: Tag) -> str:
+    def get_source_record_id(cls, xml) -> str:
```

Comment:
Missing type hint on `xml`, looks like this was copied over to the other classes

Reply:
Good catch, thanks!